Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -0,0 +1,267 @@
+category: Network Security
+commonfields:
+  id: Microsoft365Defender
+  version: -1
+configuration:
+- additionalinfo: The API Key to use for connection
+  defaultvalue: 9093c354-630a-47f1-b087-6768eb9427e6
+  display: APP ID
+  name: app_id
+  required: true
+  type: 4
+- defaultvalue: 3 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: Due to API limitation the maximum is 100
+  defaultvalue: '10'
+  display: Number of incidents for each fetch.
+  hidden: false
+  name: max_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+description: Microsoft 365 Defender is a unified pre- and post-breach enterprise defense
+  suite that natively coordinates detection, prevention, investigation, and response
+  across endpoints, identities, email, and applications to provide integrated protection
+  against sophisticated attacks.
+display: Microsoft365Defender
+name: Microsoft365Defender
+script:
+  commands:
+  - deprecated: false
+    description: Run this command to start the authorization process and follow the
+      instructions in the command results.
+    execution: false
+    name: microsoft-365-defender-auth-start
+  - deprecated: false
+    description: Run this command to complete the authorization process. Should be
+      used after running the microsoft-365-defender-auth-start command.
+    execution: false
+    name: microsoft-365-defender-auth-complete
+  - deprecated: false
+    description: Run this command if you need to rerun the authentication process.
+    execution: false
+    name: microsoft-365-defender-auth-reset
+  - deprecated: false
+    description: Tests the connectivity to the Azure SQL Management.
+    execution: false
+    name: microsoft-365-defender-auth-test
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: Categorize incidents (as Active,  Resolved or Redirected).
+      isArray: false
+      name: status
+      predefined:
+      - Active
+      - Resolved
+      - Redirected
+      required: false
+      secret: false
+    - default: false
+      description: "Owner of the incident.\t"
+      isArray: false
+      name: assigned_to
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '100'
+      description: Number of incidents in the list (Max 100).
+      isArray: false
+      name: limit
+      required: false
+      secret: false
```

Comment:
we should have an offset argument here as well

Reply:
Done